### ISSUE TITLE
Can nbgrader_config.py be automatically created by assigning the course name and folder in the Web UI?

### ISSUE THREAD

**Issue by fishfree** [jupyter/nbgrader#1010]
Now I have to firstly create a text file, writing into the config lines, then rename it to nbgrader_config.py. So it's not very intuitive for a novice user of Jupyter.

**Comment by jhamrick** [comment]
Unfortunately you'll have to provide a bit more detail here as I'm not sure which part of the UI you're referring to. If you could provide a screenshot of the part of the web UI you're referring to that would be helpful.

One thing we could do would be to have an option to run `nbgrader quickstart` from the formgrader, to initialize the `nbgrader_config.py` file. Currently if you run the formgrader from a directory without `nbgrder_config.py` you see a screen that looks like this, which I agree is not as helpful as it could be:

![image](https://user-images.githubusercontent.com/83444/46572386-c8652680-c97c-11e8-987d-40f31c6dab20.png)

**Comment by fishfree** [comment]
@jhamrick Yes, it will be wonderful to have a button on which click to execute `nbgrader quickstart` command. It will be much more intuitive.

**Comment by perllaghu** [comment]
I'm in the process of changing code so that the `course-id` is read from an environment variable - so combining this with the idea that the `course-id` is defined by some external agency.

.... dynamically changing the `nbgrader_config.py` file may be the simplest solution with the current model
